src/mapbox/mapbox.ts

```diff
@@ -862,7 +862,7 @@ export default class Mapbox {
     }
     if (eventType in cameraEvents) {
       if (typeof event === 'object') {
-        (event as ViewStateChangeEvent).viewState = transformToViewState(tr);
+        (event as unknown as ViewStateChangeEvent).viewState = transformToViewState(tr);
       }
       if (this._map.isMoving()) {
```

Comment:
Some type errors which for some reason weren't fixed on their branch. Maybe they're using a different version of ts? No idea but it was an easy fix